test/e2e/tests/settings/ipfs-ens-resolution.spec.js

```diff
@@ -10,25 +155,36 @@ describe('Settings', function () {
   it('Redirects to ENS domains when user inputs ENS into address bar', async function () {
     // Using proxy port that doesn't resolve so that the browser can error out properly
     // on the ".eth" hostname.  The proxy does too much interference with 8000.
-    const { driver } = await buildWebDriver({ proxyUrl: '127.0.0.1:8001' });
```

Comment:
to preserve the proxy number, we now pass this as an option in fixtures, and forward it to the chrome/firefox drivers as an option